airgen/src/lib.rs

```diff
@@ -80,7 +80,7 @@ pub fn compile<T: FieldElement>(input: AnalysisASMFile<T>) -> PILGraph<T> {
         .operations()
         .map(|o| Operation {
             name: MAIN_FUNCTION.to_string(),
-            id: o.id.id,
+            id: o.id.id.clone(),
```

Comment:
Need to check if we can remove those clones again later.

Reply:
FieldElement is copy, BigUint is clone. I added some other clones because I had an intermediate type during refactoring, but I think this clone is fine. Clippy detected several redundant clones, so I think it's ok.

Reply:
ok